### DISCUSSION TITLE
Where the TARO_END come from?

### DISCUSSION THREAD

**Discussion by ikeepo** [NervJS/taro#17090] [Q & A]
I use Taro 4.0.8, after build dev:weapp, I got the message "登录失败 ReferenceError: process is not defined". 
It comes from the TARO_END, but I can't find any info about this variable. It's not defined by me, where it comes from?
![image](https://github.com/user-attachments/assets/8d905b81-3e0f-4692-bcd5-5a8c37ea6997)
![image](https://github.com/user-attachments/assets/eb83c4ca-d99b-4fba-a1fb-905ba6cfbd29)
![image](https://github.com/user-attachments/assets/986f9f73-8c7f-4b87-a77b-a0afa0e37509)
![image](https://github.com/user-attachments/assets/3fc514d0-2cbb-4cc6-9f9a-ab75adfcc751)
![image](https://github.com/user-attachments/assets/19b79ef0-8eaf-4bb2-9d4e-ced91b2eaa80)

**Comment by TheKonka** [comment]
![image](https://github.com/user-attachments/assets/5f7365e2-06da-4e8e-91bc-24447cc74084)

那就是 node_modules中的包带的，关闭 `Use Exclude Settings and Ignore Files` 应该就能搜到了
